mappings/net/minecraft/world/gen/DensityFunctions.mapping

```diff
@@ -0,0 +1,445 @@
+CLASS net/minecraft/unmapped/C_mdkmcezi net/minecraft/world/gen/DensityFunctions
+	FIELD f_ezohebie DIRECT_CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_ikpluaeq MAX_REASONABLE_NOISE_VALUE D
+	FIELD f_ofybcvpy CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_zreaxxak NOISE_VALUE_CODEC Lcom/mojang/serialization/Codec;
+	METHOD m_ajxltzce blendOffset ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_akixioyj blendDensity (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 input
+	METHOD m_bavhhzle terrainShaperSpline (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_mdkmcezi$C_uejbabyt$C_zlreguhf;DD)Lnet/minecraft/unmapped/C_kzwapyud;
```

Comment:
Spline?

Reply:
https://en.wikipedia.org/wiki/Spline_(mathematics)
It's a way to define a curve, basically.